Cargo.toml

```diff
@@ -12,14 +12,15 @@ rust-version = "1.58.1"
 
 # cargo-workspaces
 [workspace.metadata.workspaces]
-version = "0.7.4"
+version = "0.7.5"
 
 [dependencies]
 anyhow = "1.0.51"
 aws-config = { version = "1.0.0", features = ["behavior-version-latest"] }
 aws-types = "1.0.0"
 aws-credential-types = "1.0.0"
 aws-sdk-s3 = "0.39.1"
+aws-smithy-types = "1.0.1"
```

Comment:
I had to move it from dev-dependencies to a general one since I introduced a variant with `aws-smithy-types` error to `LakeError` regarding the byte-stream